tests/test_blueprint.py

```diff
@@ -56,6 +56,7 @@ def test_article_with_image(self):
             b"admin.insights.ubuntu.com/wp-content/uploads", response.data
         )
         self.assertIn(b"ubuntu.com/wp-content/uploads", response.data)
+        self.assertIn(b"cloudinary", response.data)
```

Comment:
Could we test it actually does the transfromation we expect?

Reply:
I changed all tests mentioned to check for the `src` transformation of the image.

I am not checking the whole transformation, because it's a bit tedious to string check something that looks like this:
```html
<img decoding="async" src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto,w_650/https://ubuntu.com/wp-content/uploads/3b1d/dell-xps-2004.jpg" srcset="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto,w_350/https://ubuntu.com/wp-content/uploads/3b1d/dell-xps-2004.jpg 350w, https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto,w_650/https://ubuntu.com/wp-content/uploads/3b1d/dell-xps-2004.jpg 650w, https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto,w_1300/https://ubuntu.com/wp-content/uploads/3b1d/dell-xps-2004.jpg 1300w, https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto,w_1950/https://ubuntu.com/wp-content/uploads/3b1d/dell-xps-2004.jpg 1950w" sizes="(max-width: 400px) 350w, 650px" loading="lazy">
```

If you guys think I should check the whole thing, let me know.